cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/plugins/test/TestPlugin.java

```diff
@@ -35,18 +35,43 @@
 
   @Override
   public String call() throws Exception {
-    if (config.timeout % 2 == 0) {
-      return config.name;
+    if (config.timeout != null) {
+      if (config.timeout % 2 == 0) {
+        return config.host + ", " + config.aBoolean + ", " + config.aByte + ", " + config.aDouble + ", " +
```

Comment:
config.host will still have to be prepended to the string as it can be null and Joiner will throw a NPE.